Sources/RibbonKit/List/RibbonListView.swift

```diff
@@ -124,6 +131,8 @@ public class RibbonListView: UIView {
 
     #if os(iOS)
     @objc private func deviceOrientationDidChange(_ notification: Notification) {
+        let orientation = UIDevice.current.orientation
+        guard !unsupportedOrientations.contains(orientation) else { return }
```

Comment:
The idea is good, but in this case I would just omit the `unsupportedOrienations` completely and just statically check whether the orientation contains `faceUp` or `faceDown`. It doesn't make sense to reload for these orientations anyway.

Reply:
Ok I understand, I will remove unsupportedOrientations completely